python/tests/conftest.py

```diff
@@ -229,6 +229,31 @@ def s3_tmpdir() -> str:
         logging.warn("Could not delete directory: %s", s3fs_path)
 
 
+@pytest.fixture(scope="session")
```

Comment:
too slow to use the default test SparkSession, we need to do some cleanup in another pull request